### ISSUE TITLE
webtorrent command fails with "Cannot read property 'toString' of undefined"

### ISSUE THREAD

**Issue by IdanAdar** [webtorrent/webtorrent-cli#64]
When trying to download Open Office via its magnet link, the operation fails as follows.

Link: https://www.openoffice.org/distribution/p2p/magnet.html
Example:
```
webtorrent -o ~/Downloads magnet:?xt=urn:sha1:a95f45d3913b748fa91e4bb00799d69a55e063ee&dn=OOo_2.1.0_Solarisx86_install_en-US.tar.gz&xs=http://mirror.switch.ch/ftp/mirror/OpenOffice/stable/2.1.0/OOo_2.1.0_Solarisx86_install_en-US.tar.gz
```
Output:
```
UNEXPECTED ERROR: If this is a bug in WebTorrent, report it!
OPEN AN ISSUE: https://github.com/webtorrent/webtorrent-cli/issues

DEBUG INFO: webtorrent-cli 1.11.0, webtorrent 0.98.24, node v8.10.0, darwin x64, exit 1
/Users/iadar/.nvm/versions/node/v8.10.0/lib/node_modules/webtorrent-cli/node_modules/webtorrent/lib/torrent.js:229
    self._debugId = parsedTorrent.infoHash.toString('hex').substring(0, 7)
                                           ^

TypeError: Cannot read property 'toString' of undefined
    at Torrent._onTorrentId (/Users/iadar/.nvm/versions/node/v8.10.0/lib/node_modules/webtorrent-cli/node_modules/webtorrent/lib/torrent.js:229:44)
    at new Torrent (/Users/iadar/.nvm/versions/node/v8.10.0/lib/node_modules/webtorrent-cli/node_modules/webtorrent/lib/torrent.js:124:32)
    at WebTorrent.add (/Users/iadar/.nvm/versions/node/v8.10.0/lib/node_modules/webtorrent-cli/node_modules/webtorrent/index.js:260:17)
    at runDownload (/Users/iadar/.nvm/versions/node/v8.10.0/lib/node_modules/webtorrent-cli/bin/cmd.js:344:24)
    at /Users/iadar/.nvm/versions/node/v8.10.0/lib/node_modules/webtorrent-cli/bin/cmd.js:198:5
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/Users/iadar/.nvm/versions/node/v8.10.0/lib/node_modules/webtorrent-cli/bin/cmd.js:197:14)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
```

**Comment by DiegoRBaquero** [comment]
Try putting the magnet between double quotes `"`

**Comment by IdanAdar** [comment]
Tried with single and double quotes, same exception is thrown.

**Comment by alxhotel** [comment]
> magnet:?xt=urn:sha1:a95f45d3913b748fa91e4bb00799d69a55e063ee&dn=OOo_2.1.0_Solarisx86_install_en-US.tar.gz&xs=http://mirror.switch.ch/ftp/mirror/OpenOffice/stable/2.1.0/OOo_2.1.0_Solarisx86_install_en-US.tar.gz

This magnet link is used by Gnutella not Bittorrent.

Bittorrent magnet links should be of the form  `magnet:?xt=urn:btih:d218bc...`. 

We have fixed this bug in webtorrent/parse-torrent#80 (thanks @KayleePop) and now `webtorrent-cli` throws an `Invalid torrent identifier` if the magnet link is not from bittorrent.
